intersphinx_registry/lookup.py

```diff
@@ -43,7 +65,6 @@ def main():
 
     flattened = []
     for base_url, obj in urls:
-
         final_url = urljoin(base_url, obj)
 
         resp = requests.get(final_url)
```

Comment:
When you want to have fun and if you like it, try converting this to use async/await for loops with [trio](https://trio.readthedocs.io/en/stable/) and do async fetching. Yo might need `asks`instead of requests.